apisix/control/v1.lua

```diff
@@ -276,6 +277,28 @@ function _M.dump_service_info()
     return 200, info
 end
 
+function _M.dump_all_plugin_metadata()
+    local names = plugins.get_plugins_list()
```

Comment:
Better to use another way to do it. Maybe a new method in the `plugin` module or some other way.
We can't let the module in control-api require the module in admin-api. They are at the same level.